Content.Shared/_DV/CosmicCult/CosmicCult.Events.cs

```diff
@@ -0,0 +1,18 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared._DV.CosmicCult;
+
+[Serializable, NetSerializable]
+public sealed partial class CosmicSiphonIndicatorEvent : EntityEventArgs
+{
+    public NetEntity Target;
+
+    public CosmicSiphonIndicatorEvent(NetEntity target)
+    {
+        Target = target;
+    }
+
+    public CosmicSiphonIndicatorEvent() : this(new())
```

Comment:
Use primary constructor
```suggestion
public sealed partial class CosmicSiphonIndicatorEvent(NetEntity target) : EntityEventArgs
{
    public NetEntity Target = target;

    public CosmicSiphonIndicatorEvent() : this(new())
```